tor/cnd.toml

```diff
@@ -0,0 +1,34 @@
+[network]
```

Comment:
In the past, config files that we checked into source control were outdated very quickly.

What is gonna prevent this one from being outdated?

Reply:
Cool, will remove.